### PR TITLE
Set text color for dark mode skeleton

### LINKED ISSUE #8214: Black on black on dark mode in "Unable to connect to Home Assistant."
## Checklist
I couldn't do any of this, because it's hard to cause "unable to connect to home assistant".
- [ ] I have updated to the latest available Home Assistant version.
- [ ] I have cleared the cache of my browser.
- [ ] I have tried a different browser to see if it is related to my browser.

## The problem

I use dark mode in my OS. Sometimes I get a random "Unable to connect to Home Assistant." error, and it's black text on a black background.
image
This style causes the black background:
image
This bug can be fixed by changing the style to this:
image
Which results in this:
image

## Expected behavior

Expected behavior mockup in Firefox devtools:
image

## Steps to reproduce

1. Enable dark mode on your OS
2. Try to load lovelace (Firefox may be a factor)
3. Get the error (I'm not sure how)
4. See text

## Environment

- Home Assistant release with the issue: 2021.1.3
- Last working Home Assistant release (if known): N/A
- Browser and browser version: Firefox 84.0.2 (64-bit)
- Operating system: Windows

## State of relevant entities

N/A
```yaml

```

## Problem-relevant frontend configuration

I wasn't trying to load Lovelace.
```yaml

```

## Javascript errors shown in your browser console/inspector

image

```txt
contentscript.js: https://ourliltreehouse.duckdns.org/hacs <empty string> contentscript.js:24:9
BROWSER_MOD 1.2.3 IS INSTALLED
    DeviceID: a2b6a40f-9973ecc8 browser_mod.js:149:8195
Firefox can’t establish a connection to the server at wss://ourliltreehouse.duckdns.org/api/websocket. core.94ddb68d.js:1
Uncaught (in promise) 1 2
```
I have the AdNauseum browser extension (`contentscript.js`) and browser_mod installed.

### PR DESCRIPTION
**Pull request by KTibow**
## Proposed change

This sets the text color as well as the background to avoid black on black text.
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #8214
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/html/index.html.template [hunk 1]
```diff
@@ -51,6 +51,7 @@
       @media (prefers-color-scheme: dark) {
         html {
           background-color: #111111;
+          --primary-text-color: #EEEEEE !important;
```
Comment (bramkragten) [inline src/html/index.html.template:54]:
```suggestion
          --primary-text-color: #e1e1e1 !important;
```

Comment (bramkragten) [reply src/html/index.html.template:54]: I don't think we should use `!important`, will this be overruled by the user theme later?

Comment (KTibow) [reply src/html/index.html.template:54]: Oh. It will. How could I reproduce this situation, so I could find a better fix?

Comment (bramkragten) [reply src/html/index.html.template]: Run this code, set your OS mode to dark and set a theme?

Comment (KTibow) [reply src/html/index.html.template]: I mean reproduce the error loading/ connecting to HA.

Comment (bramkragten) [reply src/html/index.html.template]: Comment this: <https://github.com/home-assistant/frontend/blob/dev/src/util/init-skeleton.ts#L4>

Comment (KTibow) [reply src/html/index.html.template]: Would a better way to be to add a `style` tag to https://github.com/home-assistant/frontend/blob/dev/src/layouts/ha-init-page.ts?

Comment (bramkragten) [reply src/html/index.html.template]: Where is the `--primary-color` defined used in `ha-init-page`?

**Comment by bramkragten** [comment]
No longer needed

**Pull request closed without merging**